packages/terra-core-docs/src/terra-dev-site/doc/status-view/example/Example.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import StatusView from 'terra-status-view';
+import ToggleButton from 'terra-toggle-button';
+
+const Example = () => (
+  <ToggleButton closedButtonText="ToggleButton">
+    <StatusView
+      role="alert"
+      aria-live="polite"
+      message="Testing"
```

Comment:
```suggestion
      message="Testing"
```
Update this with meaning full text